wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/MessageHelper.kt

```diff
@@ -0,0 +1,82 @@
+package com.squareup.wire.kotlin
+
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import okio.Buffer
+import okio.ByteString
+import java.io.IOException
+
+class UnkownFieldsBuilder {
+  var unknownFieldsByteString = ByteString.EMPTY
```

Comment:
nit: rename to byteString ? Since the `unknownFields` is implied. Same for the fields below